config-skeleton.ini

```diff
@@ -1,33 +1,35 @@
 [App]
-VideoPath: /repo/data/gard1-4.mp4
+VideoPath = /repo/data/gard1-4.mp4
+Host = 0.0.0.0
+Port = 8001
+Resolution = 640,480
+Encoder = videoconvert ! video/x-raw,format=I420 ! x264enc speed-preset=ultrafast
+
+[CORE]
 Host: 0.0.0.0
-Port: 8000
-Resolution: 640,480
-Encoder: videoconvert ! video/x-raw,format=I420 ! x264enc speed-preset=ultrafast
+QueuePort: 8010
+QueueAuthKey: shibalba
 
 [Detector]
-; Supported devices: Jetson , EdgeTPU, Dummy
-Device: EdgeTPU
-; Detector's Name can be either "mobilenet_ssd_v2", "pedestrian_ssdlite_mobilenet_v2" or "pedestrian_ssdlite_mobilenet_v2"
-; the first one is trained on COCO dataset and next two are trained on Oxford Town Center dataset to detect pedestrians
-Name: pedestrian_ssdlite_mobilenet_v2
-;ImageSize should be 3 numbers seperated by commas, no spaces: 300,300,3
```

Comment:
I think we should leave these comments. I understand that the save function regenerates the whole `.ini`, do you know if this is doable with the `configparser` API? Or should we just make a config-skeleton.ini.example?

Reply:
You are right. There are solutions [here](https://stackoverflow.com/questions/6620637/writing-comments-to-files-with-configparser)